dagshub/data_engine/model/query_result.py

```diff
@@ -401,6 +404,78 @@ def download_binary_columns(
             num_proc=num_proc,
         )
 
+    def predict_with_mlflow_model(
+        self,
+        repo: str,
+        name: str,
+        version: str = "latest",
+        pre_hook: Callable = lambda x: x,
+        post_hook: Callable = lambda x: x,
+        batch_size: int = 1,
+        log_to_field: str = None,
+        return_predictions: bool = True,
```

Comment:
WDYT about getting rid of this argument completely and just always returning predictions?
It doesn't really cost anyone anything to return the resulting list in the end.

Reply:
Agreed!